web/src/main/angular/src/app/core/components/agent-statistic-list/agent-statistic-list.component.ts

```diff
@@ -88,7 +90,10 @@ export class AgentStatisticListComponent implements OnInit  {
                 field: 'agent',
                 width: 300,
                 filter: 'agTextColumnFilter',
-                tooltipField: 'agent'
+                tooltipField: 'agent',
+                cellStyle: {
+                    'cursor': 'pointer'
+                }
```

Comment:
The changes you made work well! I think it would be better to apply the same cell-style on the agent field so that users can assume the agent field is also clickable.
```suggestion
                cellStyle: {
                    color: 'rgb(54, 162, 235)',
                    'font-weight': 600,
                    'cursor': 'pointer'
                }
```
ex)
![스크린샷 2021-04-21 오전 11 24 02](https://user-images.githubusercontent.com/8830238/115487656-1a000300-a294-11eb-9d16-3a43e0d7cf06.png)